### ISSUE TITLE
Error message on successful ssh sessions completion

### ISSUE THREAD

**Issue by stevenGravy** [gravitational/teleport#21267]
Expected behavior:

Error messages should be written for genuine errors. Having an error message when there is not an error makes it hard to distinguish actual errors from noise.

Current behavior:

Each ssh session that is exitted has this error in the SSH Service.  The session successfully exited and had session recording withouth issue. 

```bash
ERRO [SESSION:N] Timed out waiting for PTY copy to finish, session data  may be missing. session_id:7a587a83-bc4b-4d8d-b1bf-bd164ee2a494 srv/sess.go:1023
```

Bug details:
- Teleport version: 11.3.2
- Recreation steps: Open a SSH session in UI or CLI.  The agent will log this after the session is completed.

**Comment by zmb3** [comment]
I'm not able to reproduce this on 11.3.2 or master.

**Comment by stevenGravy** [comment]
Seeing this with cloud and self-hosted 11.3.2 tenants on AWS EC2 instances types aws linux 2, ubuntu 22.04 and ubuntu 14.

**Comment by stevenGravy** [comment]
I do not see this on a local Teleport instance with a SSH service, only with SSH services joining from other machines.

**Comment by zmb3** [comment]
I was able to reproduce this on cloud with a 12.1.1 agent.

**Comment by zmb3** [comment]
@jakule can you have a look?

- 16:55:46: new session
- 16:56:01: leave session
- 16:56:01: `session.data` emitted
- 16:56:01: `session.leave` emitted
- 16:56:01: 5s wait timer expires

```
2023-03-17T16:55:46-04:00 INFO [SESSION:N] New party ServerContext(52.223.52.117:443->192.168.1.163:41042, user=zmb, id=1) party(id=e999bcfd-65ab-4a3c-b46e-a5075debb922) joined session session_id:10724598-75c1-4753-b8c2-3b894d46ffa5 srv/sess.go:1490
2023-03-17T16:56:01-04:00 INFO [SESSION:N] Closing party e999bcfd-65ab-4a3c-b46e-a5075debb922 srv/sess.go:1647
2023-03-17T16:56:01-04:00 INFO [SESSION:N] Removing party ServerContext(52.223.52.117:443->192.168.1.163:41042, user=zmb, id=1) party(id=e999bcfd-65ab-4a3c-b46e-a5075debb922) from session session_id:10724598-75c1-4753-b8c2-3b894d46ffa5 srv/sess.go:1295
2023-03-17T16:56:01-04:00 INFO [AUDIT]     session.data addr.remote:52.223.52.117:443 code:T2006I ei:2.147483646e+09 event:session.data login:zmb namespace:default rx:8394 server_id:ac113843-ba80-41ed-85b0-7fe6775dac03 sid:10724598-75c1-4753-b8c2-3b894d46ffa5 time:2023-03-17T20:56:01.223Z tx:8460 uid:2f1e8bec-834f-43fc-91cc-c6d4270a1624 user:zmb3 events/emitter.go:265
2023-03-17T16:56:01-04:00 INFO [AUDIT]     session.leave code:T2003I ei:53 event:session.leave login:zmb namespace:default server_id:ac113843-ba80-41ed-85b0-7fe6775dac03 sid:10724598-75c1-4753-b8c2-3b894d46ffa5 time:2023-03-17T20:56:01.223Z uid:53b6cd43-fb4f-41b0-825b-b356eefa6354 user:zmb3 events/emitter.go:265
2023-03-17T16:56:06-04:00 ERRO [SESSION:N] Timed out waiting for PTY copy to finish, session data  may be missing. session_id:10724598-75c1-4753-b8c2-3b894d46ffa5 srv/sess.go:1055
2023-03-17T16:56:06-04:00 INFO [SESSION:N] Stopping session session_id:10724598-75c1-4753-b8c2-3b894d46ffa5 srv/sess.go:635
2023-03-17T16:56:06-04:00 INFO [AUDIT]     session.end addr.remote:52.223.52.117:443 code:T2004I ei:54 enhanced_recording:false event:session.end interactive:true login:zmb namespace:default participants:[zmb3] proto:ssh server_id:ac113843-ba80-41ed-85b0-7fe6775dac03 session_recording:node session_start:2023-03-17T20:55:46.490496118Z session_stop:2023-03-17T20:56:06.186860299Z sid:10724598-75c1-4753-b8c2-3b894d46ffa5 time:2023-03-17T20:56:06.187Z uid:7dab0de2-888a-4e28-a858-18de40138a41 user:zmb3 events/emitter.go:265
2023-03-17T16:56:06-04:00 INFO [SESSION:N] Closing session session_id:10724598-75c1-4753-b8c2-3b894d46ffa5 srv/sess.go:684
```

**Comment by zmb3** [comment]
Fixed by #45879
